### PR TITLE
Fixed validation issue with decimal values with precision 0

### PR DESCRIPTION
**Pull request by fernst**

### CODE REVIEW

File: database-commons/src/test/java/io/cdap/plugin/db/CommonSchemaReaderTest.java [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.sql.Types;
+
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
```
Comment (tivv) [inline database-commons/src/test/java/io/cdap/plugin/db/CommonSchemaReaderTest.java:34]: As soon as you are using runner, it makes sense to use `@Mock` annotation. That's the main reason to use the runner

**Review by tivv — COMMENTED** [review]
nit: as soon as you are adding tests (great, thanks!), would you mind also adding a GutHub Actions build? This is a PR example: https://github.com/data-integrations/wrangler/pull/490

File: database-commons/src/test/java/io/cdap/plugin/db/CommonSchemaReaderTest.java [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+public class CommonSchemaReaderTest {
+
+  CommonSchemaReader reader;
+
+  @Mock
+  ResultSetMetaData metadata;
+
+  @Before
+  public void before() {
+    reader = new CommonSchemaReader();
```
Comment (tivv) [inline database-commons/src/test/java/io/cdap/plugin/db/CommonSchemaReaderTest.java:44]: nit: move to the field declaration, this way you save few lines of the boilerplate code

**Review by tivv — APPROVED** [review]

**Review by CuriousVini — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**